pahfit/base.py

```diff
@@ -287,8 +300,24 @@ def plot(self, ax, x, y, model):
 
         ax.plot(x, cont_y * ext_model / x, "k-")
 
-        ax.set_xlabel(r"$\lambda$ [$\mu m$]")
         ax.set_ylabel(r"$\nu F_{\nu}$")
+        ax.set_yscale("linear")
+        ax.set_xscale("log")
+
+        # residuals
+        res = (y - model(x)) / x
+        std = np.std(res)
+        ax = axs[1]
+        ax.minorticks_on()
+        ax.tick_params(which='both', right='on', top='on', direction='in')
+        ax.axhline(0, linestyle='--', color='gray', zorder=0)
+        ax.plot(x, res, color='k', zorder=1)
+
+        ax.set_ylim(-2 * std, 2 * std)
```

Comment:
It would be great if you could put in the scale factor (here 2) as a keyword input to this function (e.g., `scalefac_resid=2` in the command line) so that it can be changed by users.

Reply:
Sure! I'll try to do it sometime today or tomorrow and also update plot_pahfit.py script